### DISCUSSION TITLE
Configure DevTools hotkey

### DISCUSSION THREAD

**Discussion by sinatrocious** [AvaloniaUI/Avalonia#14457] [Ideas]
Is it possible to change `Ctrl+Shift` hotkey?

![image](https://github.com/AvaloniaUI/Avalonia/assets/91904521/17b51af3-0974-49fc-99cd-65531ab222b9)

I am using same key combination to switch languages in Windows. I would like to change it to `Shift` or maybe `Alt+Control`. Is it possible?

**Comment by timunie** [comment]
I believe it is not yet possible but a good idea.

**Reply by timunie** [reply]
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Diagnostics/Diagnostics/DevToolsOptions.cs

**Comment by stevemonaco** [comment]
No, the hotkeys are hardcoded (besides the gesture to open DevTools). [Here is the relevant code](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml.cs#L189) `MainWindow` also has `SetOptions` method which you could pull gestures from, if you want to PR to add them to `DevToolsOptions` as Tim linked. May need to be a collection (or a null fallback) because the original implementation has several variations of ctrl+shift keys.
